tap-snapshots/test-run-save-file.js-TAP.test.js

```diff
@@ -96,6 +96,8 @@ not ok 2 - a/b/f1.js # {time}
         //f1.js
           require("{CWD}/").fail('a/b')
         --^
+      stack: |
```

Comment:
I was not able to trace out why these stack's in this snapshot file are no longer completely stripped.